spec/models/document_type_spec.rb

```diff
@@ -5,13 +5,13 @@
 
   describe "all configured document types are valid" do
     it "conforms to the document type schema" do
-      document_type_schema = JSON.parse(File.read("config/schemas/document_type.json"))
+      expect(document_types).to all(match_json_schema("document_type"))
+    end
+
+    it "has locale keys that conform to the document type locale schema" do
       document_types.each do |document_type|
-        validator = JSON::Validator.fully_validate(document_type_schema, document_type)
-        expect(validator).to(
-          be_empty,
-          "Validation for #{document_type['id']} failed: \n\t#{validator.join("\n\t")}",
-        )
+        translations = I18n.t("document_types.#{document_type['id']}").deep_stringify_keys
+        expect(translations).to match_json_schema("document_types_locale")
```

Comment:
Minor: slight inconsistency between `document_type` and `document_type*s*_locale`.